src/lib/DatawrapperIframe/DatawrapperIframe.svelte

```diff
@@ -0,0 +1,62 @@
+<script>
+  import { createEventDispatcher } from "svelte";
+  import datawrapper from "./events";
+  import datawrapperEventList from "./datawrapper-event-list.json";
+
+  /*** @type {string} HTML title of the iframe */
+  export let title;
+
+  /*** @type {string} HTML aria-label of the iframe */
+  export let ariaLabel;
+
+  /*** @type {string} Datawrapper chart ID */
+  export let datawrapperId;
+
+  /*** @type {string} iframe scrolling attribute */
+  export let scrolling = "no";
+
+  /*** @type {string} iframe frameborder attribute */
+  export let frameborder = "0";
+
+  /*** @type {string} iframe height attribute (placeholder since reactivity is enabled) */
+  export let height = "500";
+
+  /*** @type {string} CSS styles of the iframe */
+  export let style = "width: 0; min-width: 100% !important; border: none;";
+
+  // add event dispatching for Datawrapper interaction events
+  // more: https://developer.datawrapper.de/docs/listening-to-chart-interaction-events#visualization-events
+  const dispatch = createEventDispatcher();
+
+  datawrapperEventList.forEach((eventName) => {
+    datawrapper.on(eventName, (e) => {
+      dispatch(eventName.replace(".", ""), e);
+    });
+  });
+
+  // responsive iframe code form Datawrapper
+  window.addEventListener("message", function (a) {
+    if (void 0 !== a.data["datawrapper-height"]) {
+      var e = document.querySelectorAll("iframe");
+      for (var t in a.data["datawrapper-height"])
+        for (var r = 0; r < e.length; r++)
+          if (e[r].contentWindow === a.source) {
+            var i = a.data["datawrapper-height"][t] + "px";
+            e[r].style.height = i;
+          }
+    }
+  });
```

Comment:
let's wrap this in a conditional check for window:
```js
if (typeof window !== "undefined") {}
```
so it doesn't bomb during ssr